### PR TITLE
[17.0][MIG]mass_mailing_resend:Migration to 17.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please don't put `[WIP]` in the title, but use draft status in the PR.

**Review by hbrunn — APPROVED** [review]

File: mass_mailing_resend/readme/CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The migration of this module from 15.0 to 16.0 was financially supported
```
Comment (pedrobaeza) [inline mass_mailing_resend/readme/CREDITS.md]: Please remove this file, as this doesn't apply for this version.

Comment (bizzappdev) [reply mass_mailing_resend/readme/CREDITS.md]: done.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**